src/helpers.ts

```diff
@@ -41,6 +41,7 @@ export function requireFromString({
   }
 
   if (typeof module === 'function') {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
Changing `args` to be of type `unknown[]` gets around this issue also but wasn't sure if I should change `RequireFromStringOptions` typings.